float64[] desiredRelPose # this can be relative desire pose
float64[] desiredAbsPose # this can be relative desire pose
float64[] currentRelPose # this can be relative desire pose
float64[] currentAbsPose # this can be relative desire pose
float64[] joint_state_left # this can be relative desire pose
float64[] joint_state_right # this can be relative desire pose
float64[] del_q # this can be relative desire pose
float64 norm_error_abs
float64 norm_error_rel
float64 timeStamp
int32 controlMode